docs/snippets/common/button-story-primary-composition.js.mdx

```diff
@@ -1,15 +1,23 @@
 ```js
 // Button.stories.js | Button.stories.jsx | Button.stories.ts | Button.stories.tsx
 
-const Primary = ButtonStory.bind({});
-Primary.args = {
-  primary: true,
-  label: 'Button',
-}
-
-const Secondary = ButtonStory.bind({});
-Secondary.args = {
-  ...Primary.args,
-  primary: false,
-}
-```
\ No newline at end of file
+import Button from './Button';
+
+export default {
+  component: Button,
+};
+
+export const Primary = {
+  args: {
+    primary: true,
+    label: 'Button',
+  },
+};
+
+const Secondary = {
+  args: {
+    ...Primary.args,
+    primary: false,
+  },
+};
```

Comment:
This is just a suggestion based on @shilman's blog post. I don't know which is considered best practice (or if it matters)
```suggestion
const Secondary = {
  ...Primary,
  args: {
    primary: false,
  },
};
```